src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py

```diff
@@ -17,7 +17,7 @@
 '''
 
 
-def get_github_access_token(cmd, scope_list=None):
+def get_github_access_token(cmd, scope_list=None):  # pylint:disable=unused-import
```

Comment:
Since you have removed import, is this change required?

Reply:
I used the wrong checker name. Fixed.